doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -552,6 +553,11 @@ Bluetooth libraries and services
   * Updated by deleting reset in progress flag from settings storage instead of storing it as ``false`` on factory reset operation.
     This is done to ensure that no Fast Pair data is left in the settings storage after the factory reset.
 
+* :ref:`bt_le_adv_prov_readme` library:
+
+  * Changed the allowed range of the :kconfig:option:`CONFIG_CONFIG_BT_ADV_PROV_FAST_PAIR_ADV_BUF_SIZE` Kconfig option and set its default value to 20.
+    This is done to align the buffer size to new Fast Pair not discoverable advertising data size.
```

Comment:
`... data size after salt size was increased from 1 byte to 2 bytes.`? (we may be even more explicit here I think